src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h

```diff
@@ -116,38 +116,38 @@ class VerletListsCells
     switch (_buildTraversal) {
       case c08: {
         if (useNewton3) {
-          auto traversal = C08Traversal<LinkedParticleCell, typename verlet_internal::VerletListGeneratorFunctor,
+          auto buildTraversal = C08Traversal<LinkedParticleCell, typename verlet_internal::VerletListGeneratorFunctor,
```

Comment:
can't you remove / simplify this switch tree using 
```
std::unique_ptr<TraversalInterface> TraversalSelector<ParticleCell>::generateTraversal()?
```
(the new one with less templates)

Reply:
can probably be replace be sth. like
```
// first check whether _buildTraversal is a valid option.
    std::set<TraversalOption> validTraversalOptions{c08, c01, c18};
    if (std::find(validTraversalOptions.begin(), validTraversalOptions.end(), _buildTraversal) ==
        validTraversalOptions.end()) {
      utils::ExceptionHandler::exception("VerletListsCells::updateVerletLists(): unsupported Traversal: {}",
                                         _buildTraversal);
    }
    if (_buildTraversal == TraversalOption::c01 and useNewton3) {
      utils::ExceptionHandler::exception("VerletListsCells::updateVerletLists(): c01 does not support newton3");
    }

    // generate buildTraversal and iterate
    auto buildTraversal = TraversalSelector<LinkedParticleCell>::generateTraversal(
        _buildTraversal, f, this->getTraversalSelectorInfo(), DataLayoutOption::aos,
        useNewton3 ? Newton3Option::enabled : Newton3Option::disabled);
    this->_linkedCells.iteratePairwise(&f, &buildTraversal);
```
but will not do that for now

Reply:
#283